src/pkg/path/service_type.go

```diff
@@ -14,10 +14,6 @@ var ErrorUnknownService = clues.New("unknown service string")
 //
 // Metadata services are not considered valid service types for resource paths
 // though they can be used for metadata paths.
-//
-// The order of the enums below can be changed, but the string representation of
```

Comment:
(just skimming cause I was curious) we should keep this comment. It documents the expectations for updating these enums (i.e. can't change string form of them)

Reply:
check, will do